integrations/gtag/lib/index.js

```diff
@@ -668,41 +667,47 @@ GTAG.prototype.timingCompletedEnhanced = function(track) {
  * @param  opt
  */
 
-function trackPageViewEvent(page, options) {
+GTAG.prototype.trackPageViewEvent = function(page, options) {
   var name = page.fullName();
   var category = page.category();
   var props = page.properties();
+  var campaign = page.proxy('context.campaign');
+  var track;
+
+  if (campaign && options.gaWebMeasurementId) {
+    push('config', options.gaWebMeasurementId, {
+      campaign: campaign
+    });
+  }
+
+  if (campaign && options.gaWebAppMeasurementId) {
+    push('config', options.gaWebAppMeasurementId, {
+      campaign: campaign
```

Comment:
Any reason we are sending the whole campaign object here and not parsing out the details like in google analytics?

Reply:
Hm, no exact reason. Good call out. Our spec actually calls the UTM keyword `term` instead of `keyword`. I'll push a commit to fix.